docs/src/components/caption.tsx

```diff
@@ -0,0 +1,14 @@
+const Caption = (props:any) => {
```

Comment:
Can we improve the prop type here?

Reply:
Yes, all of these 'children' should be set {children?: JSX.Element}